### PR TITLE
Recommendations

### PR DESCRIPTION
**Pull request by EmmaSelvig**
added text and more available cards. Will someone please have a look at the layout of the cards? :)

### CODE REVIEW

File: app/controllers/hikes_controller.rb [hunk 1]
```diff
@@ -52,7 +52,10 @@ def show
     authorize @booking
     related_difficulty = @hike.difficulty
     related_region = @hike.region
-    @related_hikes = Hike.where(difficulty: related_difficulty).or(Hike.where(region: related_region)).sample(4).&(Hike.where.not(id:@hike.id))
+    related_accomodation = @hike.accomodation_type
+
+
+@related_hikes = Hike.where(region: related_region).or(Hike.where(difficulty: related_difficulty)).or(Hike.where(accomodation_type: related_accomodation))&.where.not(id: @hike.id).sample(4)
```
Comment (vitorreis) [inline app/controllers/hikes_controller.rb:58]: Please add indentation (4 spaces/tab) for consistency on code style.

The query looks good 👍

File: app/views/hikes/index.html.erb [hunk 2]
```diff
@@ -13,6 +13,8 @@
                 <form method="post">
                   <div data-role="rangeslider">
                     <label for="price-min">400</label>
+                    <input type="range" name="price-min" id="price-min" value="200" min="400" max="800">
```
Comment (vitorreis) [inline app/views/hikes/index.html.erb]: Hi @EmmaSelvig I noticed that you haved duplicated the inputs for min price (`id="price-min"`).

The filter for max price is not working, I believe you just need to have 2 inputs with different names/ids (min/max) and make sure we use them when we do the search

File: app/views/hikes/show.html.erb [hunk 3]
```diff
@@ -121,10 +121,15 @@
...
       <% if related_hike.photos.attached? %>
         <%= link_to hike_path(related_hike), class:'text-decoration-none' do %>
-          <div class = "col-lg-12">
+          <div class = "col-lg-8">
             <div class="card-product">
               <%= cl_image_tag related_hike.photos.first.key, class: 'hike-image' %>
-              <div class="card-product-infos">
+                <div class="card-hike-infos">
+                  <div class="card-product-infos">
+            <div class="related_hike_name"><%= related_hike.name %></div>
```
Comment (vitorreis) [inline app/views/hikes/show.html.erb]: indentation reminder (give some more tabs to align with the row above)

**Review by vitorreis — COMMENTED** [review]
I think you are in the right direction, but the max price filter is not working as expected I believe.

**Comment by EmmaSelvig** [comment]
Thank you for the help @vitorreis! :-) I will take a look at all of it. 
@AnnaMoe the sliders are working when the JS code was inside the .erb but when I add in the JS file it doesn't recognise it. I tried to follow your logic with the weather feature but I still can't get it to work. Maybe you can see if it works for you please?

**Pull request merged**